src/login/loginctl.c

```diff
@@ -1454,9 +1454,9 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argv);
 
 #if 0 /// elogind adds some system commands to loginctl
-        while ((c = getopt_long(argc, argv, "hp:P:als:H:M:n:o:", options, NULL)) >= 0)
```

Comment:
This is the original upstream line, that has to remain unedited.

Reply:
Ah. I hadn't noticed that 340a89eee was from upstream and assumed '-P' was elogind-specific. My bad!